src/System.Runtime.InteropServices.RuntimeInformation/src/project.json

```diff
@@ -6,10 +6,11 @@
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
-    "System.Threading": "4.0.0"
+    "System.Threading": "4.0.0",
+    "System.Reflection.Extensions": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.1": {
+    "netstandard1.3": {
```

Comment:
Reverted these changes, 1.0 throws error, as System.Runtime.interopservices.dll is generation 1.1.0.0